test/crypto/rsa_sign_test.dart

```diff
@@ -2,7 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-@TestOn('vm')
```

Comment:
Did you actually test it on browsers? Also below.

IIRC this functionality is not exposed to the public API for browsers.